src/app/public/modules/phone-field/phone-field-adapter.service.ts

```diff
@@ -71,8 +71,8 @@ export class SkyPhoneFieldAdapterService implements OnDestroy {
     element.focus();
   }
 
-  public focusPhoneInput(): void {
-    this.focusElement(document.querySelector('.sky-phone-field-container input') as HTMLElement);
+  public focusPhoneInput(element: HTMLElement): void {
+    this.focusElement(element.querySelector('.sky-phone-field-container input') as HTMLElement);
```

Comment:
☝️ This was done to fix cases where there were multiple phone fields. Before, this method was always selecting the first one.